### PR TITLE
WIP: Issue 1565

### PR DESCRIPTION
**Pull request by prb112**
Plugin now generates output like this... 
it runs a message digest over the resources as they are created

'/xml/ibm/complete-absent/StructureMap-49.xml' occured '2' times
'/xml/ibm/minimal/RequestGroup-1.xml' occured '6' times
'/json/ibm/minimal/MedicinalProductUndesirableEffect-1.json' occured '2' times
'/json/ibm/minimal/Observation-1.json' occured '11' times
'/xml/ibm/complete-absent/Parameters-10.xml' occured '3' times
'/xml/ibm/complete-absent/StructureMap-17.xml' occured '2' times
'/xml/ibm/minimal/Group-1.xml' occured '5' times
'/json/ibm/minimal/EvidenceVariable-1.json' occured '2' times
'/xml/ibm/minimal/EventDefinition-1.xml' occured '2' times
'/json/ibm/minimal/ConceptMap-1.json' occured '2' times
'/xml/ibm/minimal/ExplanationOfBenefit-1.xml' occured '5' times
'/xml/ibm/minimal/CarePlan-1.xml' occured '3' times
'/xml/ibm/complete-absent/Parameters-15.xml' occured '4' times
'/json/ibm/minimal/ExplanationOfBenefit-1.json' occured '5' times
'/json/ibm/minimal/Patient-1.json' occured '2' times
'/xml/ibm/complete-absent/Parameters-14.xml' occured '3' times

### CODE REVIEW

File: fhir-examples/src/main/resources/ibm-json.txt [hunk 1]
```diff
@@ -9,7 +9,6 @@ OK         json/ibm/complete-absent/AdverseEvent-1.json
 OK         json/ibm/complete-absent/AllergyIntolerance-1.json
 OK         json/ibm/complete-absent/AllergyIntolerance-2.json
 OK         json/ibm/complete-absent/AllergyIntolerance-3.json
-OK         json/ibm/complete-absent/AllergyIntolerance-4.json
 OK         json/ibm/complete-absent/AllergyIntolerance-5.json
```
Comment (lmsurpre) [inline fhir-examples/src/main/resources/ibm-json.txt:12]: Is there a way to avoid the gap in numbering?

Comment (prb112) [reply fhir-examples/src/main/resources/ibm-json.txt:12]: There might be... but it means 4 is a dupe of 1,2,3

File: fhir-examples/src/main/resources/json/ibm/complete-absent/AllergyIntolerance-4.json [hunk 2]
```diff
@@ -376,7 +376,7 @@
             ]
         }
     },
-    "_onsetDateTime": {
```
Comment (lmsurpre) [inline fhir-examples/src/main/resources/json/ibm/complete-absent/AllergyIntolerance-4.json:379]: What the ExamplesGenerator is *supposed* to be doing is creating a variant of each resource type with each choice data type.  AllergyIntolerance.onsetDateTime is a choice of 5 different types, and so I'd expect there to be 5 instances, one for each choice type:

onsetDateTime |   |   | dateTime |  
-- | -- | -- | -- | --
onsetAge |   |   | Age |  
onsetPeriod |   |   | Period |  
onsetRange |   |   | Range |  
onsetString |   |   | string

Comment (prb112) [reply fhir-examples/src/main/resources/json/ibm/complete-absent/AllergyIntolerance-4.json:379]: it was accounting for duplicates in each type... meaning mock has duplicate in absent, minimal and they were deduping each other.  I have now accounted for it.

**Comment by prb112** [comment]
I'll create a new PR in the next few days.

**Pull request closed without merging**